### PR TITLE
fixing behavior of buttons in AddedPane

### LINKED ISSUE #693: Buttons in Added Pane behave differently than in Course Pane
The buttons in the Course pane are absolutely positioned, meaning that they will remain in the same place (on the screen) even when scrolled down. This is not the same in Added pane.

The behavior should be the same to maintain consistency; Best thing to do is to probably absolutely position the Added pane buttons as well.

### PR DESCRIPTION
**Pull request by srukelman**
## Summary
set button box position to absolute
added padding to box below to prevent overlap before scrolling
## Disclaimer
i think a less hacky solution would require extensive refactoring
## Issues
closes #693

### CODE REVIEW

**Review by EricPedley — APPROVED** [review]
lgtm, @KevinWu098 do you have anything to add before merging since you self-requested review?

also, @srukelman FYI if you put the issue number in the PR description, it auto-closes it for us once we merge the PR. It's good practice to follow if you plan on doing more PR's.

File: apps/antalmanac/src/components/RightPane/AddedCourses/AddedCoursePane.tsx [hunk 1]
```diff
@@ -296,13 +296,12 @@ function AddedSectionsGrid() {
...
         <Box display="flex" flexDirection="column" gap={1}>
-            <Box display="flex" width={1}>
+            <Box display="flex" width={1} position="absolute" zIndex="2">
                 <CopyScheduleButton />
                 <ClearScheduleButton />
                 <ColumnToggleButton />
             </Box>
-
-            <Box>
+            <Box style={{ marginTop: 45 }}>
```
Comment (KevinWu098) [inline apps/antalmanac/src/components/RightPane/AddedCourses/AddedCoursePane.tsx]:
```suggestion
            <Box style={{ marginTop: 50 }}>
```
The course pane's button row takes up a space of 50px, so we should keep that consistent here.

**Review by KevinWu098 — CHANGES_REQUESTED** [review]
Just the one comment, otherwise LGTM

**Comment by KevinWu098** [comment]
> lgtm, @KevinWu098 do you have anything to add before merging since you self-requested review?
> 
> also, @srukelman FYI if you put the issue number in the PR description, it auto-closes it for us once we merge the PR. It's good practice to follow if you plan on doing more PR's.

Yeah 😅, I noticed a slight styling inconsistency but I forgot to write up the review. Review is up now ^^

**Comment by srukelman** [comment]
requested changes have been made

**Review by KevinWu098 — APPROVED** [review]
LGTM; Thanks for contributing :)) 💖

**Pull request merged**